var/spack/repos/builtin/packages/charliecloud/package.py

```diff
@@ -17,15 +17,22 @@ class Charliecloud(AutotoolsPackage):
     version('master', branch='master')
     version('0.15',   sha256='2163420d43c934151c4f44a188313bdb7f79e576d5a86ba64b9ea45f784b9921')
     version('0.14',   sha256='4ae23c2d6442949e16902f9d5604dbd1d6059aeb5dd461b11fc5c74d49dcb194')
+    version('0.13',   sha256='5740bff6e410ca99484c1bdf3dbe834c0f753c846d55c19d6162967a3e2718e0')
 
     depends_on('m4',       type='build')
     depends_on('autoconf', type='build')
     depends_on('automake', type='build')
     depends_on('libtool',  type='build')
 
-    depends_on('python@3.5:',    type='run')
-    depends_on('py-lark-parser', type='run')
-    depends_on('py-requests',    type='run')
+    # Use legacy build system up to version 0.13
```

Comment:
Our legacy build system included manual make build process. Recommend something a little more explicit. Perhaps the following.

```suggestion
    # Use skopeo and umoci for older ch-grow version dependencies
```